assets/sass/components/user-input/googlesitekit-user-input-module.scss

```
@@ -64,4 +65,56 @@
 		line-height: $lh-body-sm;
 		margin: 0;
 	}
+
+	.googlesitekit-user-input__footer {
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+
+		@media (min-width: $bp-desktop) {
+			align-items: center;
+			border-top: 1px solid $c-mercury;
+			flex-direction: row;
```

Another minor point but I'd suggest `row-reverse` here would make things a bit clearer than specifying `order: 2` below.
```suggestion
			flex-direction: row-reverse;
```

---

Updated, thank you!